documents/operating.md

```diff
@@ -0,0 +1,329 @@
+Bad Idea Factory is a Limited Liability Corporation, which means that
+partnership can't ~just~ be about the spirit or the lulz -- partners
+have legal responsibilities, ownership, and control over the company
+direction.
+
+Corporate Overlords are expected to...
+
+1. Make a capital contribution upon joining.
+2. Participate and vote in monthly plotting sessions.
+3. Participate in reviews for changes to corporate GitHub.
+4. Contribute original poetry on corporate GitHub.
+5. Stay involved and aware of the projects being undertaken by members.
+6. Help maintain and further the BIFFUD public footprint.
+7. Try to attend BIFFUD events.
+
+### Elected Roles
+In addition to base expectations, the following positions are to be held
+by corporate overlords.  They are elected annually:
+
+- **President:** Has a gavel.  The President is responsible for merging
+approved pull requests to `\corporate`.
+
+- **Vice President:** Has a smaller gavel.  The Vice President is responsible
+for laughing whenever the President makes a joke.
+
+- **Treasurer:** Keeps track of money, works with accountants, oversees
+and approves small expenses.
+
+- **Secretary:** Ensures notes and minutes are taken for meetings,
+ensures that proper CONTRIBUTING.md protocols are followed in `/corporate`.
+
+- **Beastmaster:** Ensures that the corporate overlords are aware of the
+progress and activity of BIFFUD projects.
+
+These roles are responsible for overseeing their duties, though in some
+cases that responsibility can take the form of setting up processes and
+support networks.  For instance, the Beastmaster can delegate oversight
+over a subset of BIFFUD projects if we have too many.
+
+### Adding Corporate Overlords
+Although previously thought impossible, an ancient ritual found buried
+in the ruins of the Lawyer Cave describes the process of creating new
+BIFFUD Corporate Overlords.
+
+Any Corporate Overlord may nominate the addition of a new Overlord during
+a plotting session.  If the nomination is seconded, the ritual begins.
+
+1. The Corporate Overlords are presented with a biography of the candidate.
+2. The candidate submits a piece of original poetry or other creative work
+for the Overlords' enjoyment and consideration.
+3. A vote is held.
+
+If 70% or more of the Corporate Overlords approve the addition, then after the
+plotting session ends the new Corporate Overlord is onboarded and ownership of
+the company is redistributed so that each overlord owns an equal share.
+
+
+### Removing Corporate Overlords
+A Corporate Overlord can recuse itself from BIFFUD at any time by dying, uploading
+its consciousness to the cloud, or submitting a letter of resignation to both
+the President of BIFFUD and the President of the United States.
+
+Corporate Overlords can also vote to remove a peer from the organization if
+any of the following conditions apply:
+
+- The Corporate Overlord fails to attend 50% of the monthly plotting sessions
```

Comment:
Absolutely -- the way this is done in most orgs I've been a part of is that meetings are routine (e.g. "third Wednesday of every month") -- and basic meeting schedule might even be written in the bylaws.  It is then the responsibility of members to ensure they map their schedule into being able to attend.